ortho_seq_code/gui/gui.py

```diff
@@ -111,10 +111,7 @@ def openPrecompFolder(self):
         options = QFileDialog.Options()
         options |= QFileDialog.ShowDirsOnly
         precomp_dir = QFileDialog.getExistingDirectory(
-            self,
-            "Open folder with precomputed file:",
-            "",
-            options=options,
+            self, "Open folder with precomputed file:", "", options=options,
         )
         print(precomp_dir)
```

Comment:
at some point introducing logging module to write the debug output to a log file could be useful for this repo